### PR TITLE
Remove obsolete FIXMEs and TODOs

### PR DESCRIPTION
**Pull request by m-g-sonar**

### CODE REVIEW

File: java-symbolic-execution/src/test/files/se/FlowComputationRelSV.java [hunk 1]
```diff
@@ -58,7 +58,6 @@ void npe1(Object a) {
 
   void npe2(Object a, Object b) {
     if (a == b) {
-      // FIXME SONARJAVA-2272
```
Comment (quentin-jaquier-sonarsource) [inline java-symbolic-execution/src/test/files/se/FlowComputationRelSV.java:61]: The situation is not fixed, it should refer to https://jira.sonarsource.com/browse/SONARJAVA-2303 now.

File: java-frontend/src/main/java/org/sonar/java/model/InternalSyntaxTrivia.java [hunk 2]
```diff
@@ -63,7 +63,6 @@ public boolean isLeaf() {
 
   @Override
   public void accept(TreeVisitor visitor) {
-    //FIXME do nothing
```
Comment (quentin-jaquier-sonarsource) [inline java-frontend/src/main/java/org/sonar/java/model/InternalSyntaxTrivia.java:66]: Maybe we should keep the comment, without the `FIXME`, to be explicit that this method does nothing on purpose? Same in `InternalSyntaxToken.java`.

File: java-symbolic-execution/src/test/files/se/FlowComputationRelSV.java (hunk 1 above)
Comment (m-g-sonar) [reply java-symbolic-execution/src/test/files/se/FlowComputationRelSV.java:61]: missed this while reading SONARJASVA-2272

**Review by quentin-jaquier-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**